cafebabel/core/mixins.py

```diff
@@ -17,7 +19,15 @@ def get_images_path(self):
     @property
     def image_url(self):
         if self.has_image:
-            return f'{self.get_images_url()}/{self.get_id()}'
+            return url_for('static',
+                           filename=f'{self.get_images_url()}/{self.get_id()}')
+
+    @property
+    def image_full_url(self):
```

Comment:
I would rename it to `image_absolute_url` if we had to do this that way for some reason.
However I'd suggest to use a `absolute()` template function instead.

Reply:
Please remove given that you workaround it.